feature-libs/product-configurator/rulebased/components/overview-filter/configurator-overview-filter.component.ts

```diff
@@ -44,6 +56,35 @@ export class ConfiguratorOverviewFilterComponent implements OnChanges {
     this.configuratorCommonsService.updateConfigurationOverview(inputConfig);
   }
 
+  /**
+   * Verifies whether a product is a variant product in the display only view.
+   *
+   * @returns - if `baseProduct` property of the current product is defined
+   * and provides the product code of the base product,
+   * and the current product is in the display only view
+   * then returns `true`, otherwise `false`.
```

Comment:
The JS doc is not entirely true. Because for editable configurations, the method will not return false, it will never emit. So either change the method (which would be probably better understandable) or adopt the comment.